pandas/core/arrays/integer.py

```diff
@@ -280,6 +280,8 @@ def _coerce_to_ndarray(self):
         data[self._mask] = self._na_value
         return data
 
+    __array_priority__ = 1  # higher than ndarray so ops dispatch to us
```

Comment:
Index, Series, and now DataFrame all set this to 1000.  Does differing from those matter?